ddtrace/internal/writer.py

```diff
@@ -111,6 +111,7 @@ def get_json(self):
                 # This typically happens when using a priority-sampling enabled
                 # library with an outdated agent. It still works, but priority sampling
                 # will probably send too many traces, so the next step is to upgrade agent.
+                # This also happens when DATADOG_PRIORITY_SAMPLING=false is set
```

Comment:
do we want to adjust our logging to _not_ log when that config is set or log a different message?

Reply:
Yeah I thought of that too but wasn't sure. My thought was that if someone was wondering about this, they would look in the tracer code and find this comment. Let me know if you'd like to go with one of the other options though and I'll do that instead.

Reply:
Probably not, most customers will probably just open a support case rather than looking into the code, so would be better to update the log message (or again, maybe we remove the log message? 🤷🏻 )

Reply:
@ZStriker19 @brettlangdon do we ever recommend setting `DATADOG_PRIORITY_SAMPLING=false`? If so, in what cases?

Oddly, we only support the environment variable when using `ddtrace-run` 🤔 .

> most customers will probably just open a support case rather than looking into the code

A lot of our users look at the code too

Reply:
I have never recommended and was not able to get the use case of the customer who did use it.

Reply:
I changed the log line.